tests/dialects/test_vector.py

```diff
@@ -461,7 +461,7 @@ def test_vector_masked_load_verify_mask_vector_rank():
 
     with pytest.raises(Exception) as exc_info:
         maskedload.verify()
-    assert exc_info.value.args[0] == "Expected a rank 1 mask vector."
+    assert "Expected vector rank to be 1, got 2." in exc_info.value.args[0]
```

Comment:
In terms of error messages, it might be cool to be able to report which exact field threw the error. So similar to what you had before, where you mentioned the mask vector in this example. This might however require deep changes in how verification is done within irdl. Thoughts? @math-fehr

Reply:
I agree, it would certainly be helpful in that case. I am not sure how `llvm` deals with this though, [this](https://github.com/llvm/llvm-project/blob/8d92a8226eb71f0199f4cc83b961bf5440a1e503/mlir/include/mlir/Dialect/Vector/IR/VectorOps.td#L1766) doesn't seem to be providing any operation/call-site info directly.

Reply:
Yes, I can try to make this better in IRDL